OpenSim/Simulation/Manager/Manager.h

```diff
@@ -240,13 +251,15 @@ class OSIMSIMULATION_API Manager
     * finalTime = 10.0;
     * int n = int(round(finalTime/dTime));
     * state.setTime(0.0);
+    * manager.initialize(state);
     * for (int i = 1; i <= n; ++i) {
-    *     manager.integrate(state, i*dTime);
+    *     manager.integrate(i*dTime);
     * }
```

Comment:
Consider adding an example here related to something @aseth1 said elsewhere in this review: "If you must update states or controls in a loop, you must recreate the manager within the loop (such discontinuous changes are considered "events" and cannot be handled during integration of the otherwise continuous system): <example> ...The proper way to handle this situation is to create a SimTK::EventHandler."

If you adopt @aseth1's suggestion of allowing repeated calls to `initialize()`, then you could change "you must recreate the manager within the loop" to "you must call `initialize()` within the loop".